docs/source/user-guide/compatibility.md

```diff
@@ -73,76 +73,77 @@ Spark.
 The following cast operations are generally compatible with Spark except for the differences noted here.
 
 | From Type | To Type | Notes |
-|-|-|-|
-| boolean | byte |  |
-| boolean | short |  |
+|-|---------|-|
```

Comment:
nit: Just checking whether this change is due to the changes in the producing method
Should be automatically created by `make release`